{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/careers/models.py

```diff
@@ -76,6 +120,76 @@ class Career(PageBase):
         blank=True,
     )
 
+    date_posted = models.DateField(
+        auto_now_add=True,
+        blank=True,
+        null=True,
+    )
+
+    # Schema Fields
+    employment_type = models.CharField(
+        choices=[
+            ('FULL_TIME', 'Full time'),
+            ('PART_TIME', 'Part time'),
+            ('CONTRACTOR', 'Contract'),
+            ('TEMPORARY', 'Temporary'),
+            ('INTERN', 'Internship'),
+        ],
+        max_length=16,
+        blank=True,
+        null=True,
+    )
+
+    education_requirements = models.CharField(
+        max_length=512,
```

Comment:
You should usually use a TextField for very long CharFields if you are expecting them to be this long.